documentation/book/assembly-kafka-cluster.adoc

```diff
@@ -9,18 +9,33 @@
 [id='kafka-cluster-{context}']
 = Kafka cluster
 
-When installing Kafka, {ProductName} also installs a Zookeeper cluster and adds the necessary configuration to connect Kafka with Zookeeper.
+You can use {ProductName} to deploy ephemeral and persistent Kafka clusters to {ProductPlatformName}.
 
-{ProductName} provides two options for Kafka cluster deployment:
+Ephemeral:: An ephemeral Kafka cluster is only suitable for development and testing purposes, not for production. This deployment uses `emptyDir` volumes for storing broker information (for Zookeeper) and topics or partitions (for Kafka). Using an `emptyDir` volume means that its content is strictly related to the pod life cycle and is deleted when the pod goes down.
```

Comment:
Saying "only" is not accurate imho. You can deploy an ephemeral cluster and then relying on Kafka replication to avoid data loss; it means that even a pod dies and the related volume is deleted, the broker will catch up with other brokers syncing with them as follower and getting all the messages.

Reply:
Great feedback Paolo. Remember, our aim is to document the _primary use cases_ for AMQ Streams rather than every possible technical scenario. I've added "In general" to the start of the sentence and removed "only" as you suggest.

Does the above scenario constitute a user story that we should document? If so, let's create an issue for it. :)